.github/workflows/sync_github_project.yml

```
@@ -0,0 +1,240 @@
+      - name: Sync roadmap and tasks to GitHub
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          python scripts/sync_to_github.py
+      - name: Commit tasks.json changes (if any)
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: 'chore(sync): update tasks.json with GitHub links'
+          file_pattern: collaboration/state/tasks.json
+      - name: Add issues to Project (if configured)
+        if: ${{ secrets.PROJECT_URL != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)